jax/api.py

```diff
@@ -242,22 +248,29 @@ def xla_computation(fun: Callable,
 
   For example:
 
+  >>> import jax
+  >>>
   >>> def f(x): return jax.numpy.sin(jax.numpy.cos(x))
   >>> c = jax.xla_computation(f)(3.)
   >>> print(c.GetHloText())
-  HloModule jaxpr_computation__4.5
-  ENTRY jaxpr_computation__4.5 {
-    tuple.1 = () tuple()
-    parameter.2 = f32[] parameter(0)
-    cosine.3 = f32[] cosine(parameter.2)
-    ROOT sine.4 = f32[] sine(cosine.3)
+  HloModule xla_computation_f.6
```

Comment:
I am wondering if the HLO names are deterministic.

Reply:
I think actually they are not, at least I think if you change the order in which different docs are executed you get different outputs, so I will disable doctests on the HLO outputs.

Reply:
I think they get uniquified with a counter to avoid name collisions, which XLA doesn't like.